src/modules/Credentials.js

```diff
@@ -0,0 +1,27 @@
+import store from '../state/store/configureStore';
+
+const Credentials = {
+  getFormInput(event) {
+    return {
+      email: event.target.username.value,
+      password: event.target.password.value,
+    };
+  },
```

Comment:
Should include `source: 'admin-system` :D

Reply:
done